test_cases.md

```diff
@@ -0,0 +1,107 @@
+# List of testing cases before release
+
+### Accuracy check via simulation: IBIS workflow
+
+
+**Description:**
+
+We are checking accuracy of IBIS workflow with `simulate` command. 
+It will take roughly an hour. 
+
+**Command:**
+
+```
+docker run --rm --privileged -it -v /media/data1/relatives:/media -v /etc/localtime:/etc/localtime:ro genx_relatives:latest launcher.py simulate --ref-directory /media/big_ref --cores 8 --directory /media/runs/sim-ibis-small --flow ibis --assembly hg37 --real-run
```

Comment:
May it's better to write something about `/media/data1/relative`, `/media/big_ref` directories. What should we store there? How to get reference data?